Automattic/wp-calypso
# Untangling: polish UI components of the new IA

Fixes https://github.com/Automattic/dotcom-forge/issues/9865

## Proposed Changes

This PR does the following:

- Promotes `<Panel />` components to `client/components` so that they can be reused in other global view pages like Domains
- Adds max-width: 720px to the following pages:
  - Settings -> {all subtabs}
  - Advanced Tools -> {SFTP, Database}
- Resolves the upsell positioning issue (p1732287115632309-slack-CRWCHQGUB)
- Resolves the Site tools positioning issue in production (https://github.com/Automattic/wp-calypso/pull/96488#pullrequestreview-2456409701)

> [!NOTE]  
> Sorry for the large number of changed files; it's mostly import changes 😭 


## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites.
2. Click a site.
3. Click around the subtabs and verify there are no visual issues.
4. Try with several site types.
5. Test also with `?flags=-untangling/hosting-menu` flag turned offf.

## Pre-merge Checklist
